packages/framework-provider-azure-infrastructure/src/infrastructure/arm-templates/cosmos-db-account.ts

```diff
@@ -46,7 +46,7 @@ export const template = {
         ipRangeFilter: '',
         dependsOn: [],
         capabilities: [],
-        enableFreeTier: true,
```

Comment:
As context we needed to do this because we can only have one Cosmos DB account with free tier but since we have one account per booster app we can't deploy more than one booster app so we needed to change this in order to test in parallel